tutorials/CreatingCustomArguments.md

```diff
@@ -0,0 +1,81 @@
+One of the best features from Klasa's (and Komada's) usage is that you can extend its functionality by adding new types, to achieve that, we will use our {@link Extendable}s. We will create an empty extendable, called `emoji.js`, and aim it to Klasa:
```

Comment:
You should note that the following argument type is already included in klasa, and is just an explanation of how it works.